google-map-search.html

```diff
@@ -166,6 +173,21 @@
       }
     },
 
+    /**
+     * Fetches details for a given place.
+     * @param {String} placeId The place id.
+     * @return {Promise} place The place information.
+     */
+    getDetails: function(placeId) {
+      var places = new google.maps.places.PlacesService(this.map);
```

Comment:
do you want to cache the `PlacesService` after creation? Not sure of the overhead for them.